.github/actions/javascript/postTestBuildComment/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Mark Pull Requests as Deployed"
+description: "Mark pull requests as deployed on production or staging"
+inputs:
+    PR_NUMBER:
+        description: "Pull request number"
+        required: true
+    GITHUB_TOKEN:
+        description: "Github token for authentication"
+        required: true
+        default: "${{ github.token }}"
```

Comment:
I see we have `required: true` and a default here, which doesn't really make sense.